docs/software/uenv/index.md

```diff
@@ -601,3 +601,66 @@ uenv image find @'*'%gh200
 
 !!! note
     The wild card `*` used for "all systems" must always be escaped in single quotes: `@'*'`.
+
+[](){#ref-uenv-customenv}
+## Custom environments
+
+!!! warning "[Keep bashrc clean][ref-guides-terminal-bashrc]"
```

Comment:
Is this supposed to be a standalone warning (i.e. no indentation afterwards)?

Reply:
yes